ixpe_intro.ipynb

```diff
@@ -0,0 +1,731 @@
+    "\n",
+    "In this *IXPE* example, it is not necessary to reprocess the data. Instead the level 2 data products can be analysed directly. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a8b976c9-ccfa-444e-ab8b-3bf7b39bf3dd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import heasoftpy as hsp\n",
+    "\n",
+    "# set some input\n",
+    "indir  = paths[0]\n",
+    "obsid  = indir.split('/')[-1] \n",
+    "\n",
+    "filelist = glob.glob(f'{paths[0]}/event_l2/*')\n",
+    "filelist"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ccc29af0-85b4-4d57-b841-ba2dd517b0db",
+   "metadata": {},
+   "source": [
+    "We see that there are three files: one event file for each detector. We can examine the structure of these level 2 files."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "285be5fb-f267-4cd2-b080-0319ef12fec1",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "det1_fits = filelist[0]\n",
+    "det2_fits = filelist[1]\n",
+    "det3_fits = filelist[2]\n",
+    "\n",
+    "#print the file structure for event 1 dectector file\n",
+    "out = hsp.fstruct(infile=det1_fits).stdout\n",
+    "print(out)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7303a643-bfa6-4bc9-8cb7-2c05e11e21c2",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "## Extracting the spectro polarimetric data \n",
+    "\n",
+    "### Defining the source and background regions\n",
+    "\n",
+    "To obtain the source and background spectra from the Level 2 files, we need to define a source region and background region for the extraction. \n",
+    "\n",
+    "For the source, we extract a 60\" circle centered on the source. For the background region, we use an annulus with inner radius of 132.000\" and outer radius 252.000\"\n",
+    "\n",
+    "The region files should be independently defined for each telescope; in this example, the source location has the same celestial coordinates within 0.25\" for all three detectors so a single source and a single background region can be used."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a450f537-db5d-46fb-8b9c-c65608dadec7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "f = open(\"src.reg\", \"w\")\n",
+    "f.write('circle(16:53:51.766,+39:45:44.41,60.000\")')\n",
+    "f.close()\n",
+    "\n",
+    "f = open(\"bkg.reg\", \"w\")\n",
+    "f.write('annulus(16:53:51.766,+39:45:44.41,132.000\",252.000\")')\n",
+    "f.close()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e11f067-8bbf-4f32-9a41-5bdbd42218a8",
+   "metadata": {},
+   "source": [
+    "For more information on how to define region files interactively usin [jdaviz](https://jdaviz.readthedocs.io/en/latest/), see our [jdaviz notebook](https://github.com/HEASARC/sciserver_cookbooks/blob/main/jdaviz-demo.ipynb)."
+   ]
```

Comment:
For the link to jdaviz demo to work here and on sciserver, it will need to be relative:
`[jdaviz notebook](jdaviz-demo.ipynb).`